--- origsrc/digikam-1.0.0/libs/3rdparty/libpgf/PGFplatform.h	2009-12-21 03:20:57.000000000 -0600
+++ src/digikam-1.0.0/libs/3rdparty/libpgf/PGFplatform.h	2010-01-10 02:55:00.338728000 -0600
@@ -306,7 +306,7 @@ inline OSError SetFPos(HANDLE hFile, int
 //-------------------------------------------------------------------------------
 // NetBSD
 //-------------------------------------------------------------------------------
-#ifdef __NetBSD__
+#if defined(__NetBSD__) || defined(__CYGWIN__)
 #ifndef __POSIX__ 
 #define __POSIX__ 
 #endif 
