--- origsrc/digikam-1.0.0-beta4/libs/3rdparty/libpgf/PGFplatform.h	2009-08-30 14:20:20.000000000 -0500
+++ src/digikam-1.0.0-beta4/libs/3rdparty/libpgf/PGFplatform.h	2009-09-23 03:15:33.783352600 -0500
@@ -304,7 +304,7 @@ inline OSError SetFPos(HANDLE hFile, int
 //-------------------------------------------------------------------------------
 // POSIX *NIXes
 //-------------------------------------------------------------------------------
-#ifdef __NetBSD__
+#if defined(__NetBSD__) || defined(__CYGWIN__)
 #ifndef __POSIX__
 #define __POSIX__
 #endif
