--- origsrc/digikam-2.0.0/core/digikam/CMakeLists.txt	2011-07-27 18:28:25.000000000 -0500
+++ src/digikam-2.0.0/core/digikam/CMakeLists.txt	2011-08-10 21:39:25.481342000 -0500
@@ -260,6 +260,12 @@ KDE4_ADD_APP_ICON(digikam_SRCS "../data/
 KDE4_ADD_EXECUTABLE(digikam ${digikam_SRCS})
 
 TARGET_LINK_LIBRARIES(digikam
+        advancedrename
+        queuemanager
+        cameragui
+        digikamcore
+        digikamdatabase
+
         ${QT_QTSQL_LIBRARY}
         ${QT_QT3SUPPORT_LIBRARY}
 
@@ -279,12 +285,6 @@ TARGET_LINK_LIBRARIES(digikam
         ${STRIGI_LIBRARIES}
         ${LCMS_LIBRARIES}
         ${GPHOTO2_LIBRARIES}
-
-        digikamcore
-        digikamdatabase
-        advancedrename
-        queuemanager
-        cameragui
 )
 
 IF (NOT WIN32 AND GPHOTO2_FOUND)
