--- origsrc/digikam-2.1.1/core/digikam/CMakeLists.txt	2011-09-14 01:22:02.000000000 -0500
+++ src/digikam-2.1.1/core/digikam/CMakeLists.txt	2011-09-19 19:36:05.628014900 -0500
@@ -261,6 +261,12 @@ KDE4_ADD_APP_ICON(digikam_SRCS "../data/
 KDE4_ADD_EXECUTABLE(digikam ${digikam_SRCS})
 
 TARGET_LINK_LIBRARIES(digikam
+        advancedrename
+        queuemanager
+        cameragui
+        digikamdatabase
+        digikamcore
+
         ${QT_QTSQL_LIBRARY}
         ${QT_QT3SUPPORT_LIBRARY}
 
@@ -280,12 +286,6 @@ TARGET_LINK_LIBRARIES(digikam
         ${STRIGI_LIBRARIES}
         ${LCMS_LIBRARIES}
 
-        digikamcore
-        digikamdatabase
-        advancedrename
-        queuemanager
-        cameragui
-
         # See B.K.O #258931: libgphoto2 library must be the last arg for linker.
         ${GPHOTO2_LIBRARIES}
 )
