https://bugs.kde.org/show_bug.cgi?id=290496

--- origsrc/digikam-2.5.0/core/utilities/setup/setupplugins.cpp	2012-01-02 18:32:01.000000000 -0600
+++ src/digikam-2.5.0/core/utilities/setup/setupplugins.cpp	2012-01-25 12:36:32.238999400 -0600
@@ -100,6 +100,10 @@ SetupPlugins::SetupPlugins(QWidget* pare
 
     panel->setLayout(mainLayout);
 
+#if KIPI_VERSION < 0x010400
+    d->checkAllBtn->setVisible(false);
+    d->clearBtn->setVisible(false);
+#endif
     initPlugins();
 
     // --------------------------------------------------------
@@ -158,14 +162,18 @@ void SetupPlugins::applyPlugins()
 void SetupPlugins::slotCheckAll()
 {
     QApplication::setOverrideCursor(Qt::WaitCursor);
+#if KIPI_VERSION >= 0x010400
     d->kipiConfig->slotCheckAll();
+#endif
     QApplication::restoreOverrideCursor();
 }
 
 void SetupPlugins::slotClear()
 {
     QApplication::setOverrideCursor(Qt::WaitCursor);
+#if KIPI_VERSION >= 0x010400
     d->kipiConfig->slotClear();
+#endif
     QApplication::restoreOverrideCursor();
 }
 }  // namespace Digikam
